assets/js/services/buySell.service.js

```diff
@@ -154,6 +155,17 @@ function buySell ($rootScope, $timeout, $q, $state, $uibModal, $uibModalStack, W
       });
   }
 
+  function cancelTrade (trade) {
+    let msg = 'CONFIRM_CANCEL_TRADE';
+    if (trade.medium === 'bank') msg = 'CONFIRM_CANCEL_BANK_TRADE';
+
+    return Alerts.confirm(msg, {
+      action: 'CANCEL_TRADE',
+      cancel: 'GO_BACK'
+    }).then(() => trade.cancel().then(() => service.fetchProfile()), () => {})
+      .catch((e) => { debugger; Alerts.displayError('ERROR_TRADE_CANCEL'); });
```

Comment:
This debugger was removed in https://github.com/blockchain/My-Wallet-V3-Frontend/pull/917/commits/50f2d023501a24610082954e5ead9fca0d174a30